### ISSUE TITLE
Is this project maintained?

### ISSUE THREAD

**Issue by indera** [freemed/freemed#20]
Hello freemed team!
I just wanted to get an idea of how many users of this project you know of and is it maintained?

I see that you have old Multum-NDC mapping information, is there a way to update that?

**Comment by jbuchbinder** [comment]
Yes, there is a way to do that ; the main drug database has to be updated to work with another freely-available system. Multum stopped support of their database, but IIRC, the FDA NDC database can still be used.
